spec/rspec/rails/example/rails_example_group_spec.rb

```diff
@@ -32,5 +36,18 @@ module RSpec::Rails
 
       expect(results).to all be true
     end
+
+    describe 'CurrentAttributes', order: :defined, if: ::Rails::VERSION::MAJOR >= 7 do
```

Comment:
The concept is hard to absorb at first, but here there are two types of groups, the first that is part of our spec suite, that is first evaluated when you run `rspec` over rspec-rails specs, but before running any specs. The second is defined during specs runtime, like `RSpec.describe` you can see above. I wouldn’t mix in RailsExampleGroup into the former, but it can be freely done with the latter. See example above for the reference.

Reply:
@pirj I didn't get to this in time. Seeing Jon's update, I'm just not familiar with how specs are written here and struggled a bit. I'll know how to better approach it in the future. Thanks!